<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Examples</class>
 <widget class="QDialog" name="Examples">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Examples</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:rgb(255, 245, 255)
}</string>
  </property>
  <widget class="QLabel" name="showTop">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>400</width>
     <height>170</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>170</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>170</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background: rgb(255, 255, 255)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="showBottom">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>400</width>
     <height>230</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>230</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>230</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background: rgb(255, 255, 255)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>401</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="changeBottomButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(255, 230, 255);
	border:1px solid rgb(255, 200, 255);
}
QPushButton::hover{
	background-color:rgb(255, 210, 255);
}</string>
      </property>
      <property name="text">
       <string>изменить низ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="changeTopButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(255, 230, 255);
	border:1px solid rgb(255, 200, 255);
}
QPushButton::hover{
	background-color:rgb(255, 210, 255);
}</string>
      </property>
      <property name="text">
       <string>изменить верх</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
